canonical/canonical.com
# Fix individual blog posts tagged lang:kr

## Done

Regression of https://warthogs.atlassian.net/browse/WD-5635

## QA

- Check /blog/ubuntu-core-an-immutable-linux-desktop-kr is now accessible
- Check /blog/tag/lang:kr still works
- Check https://canonical.com/blog?page=3 doesn't show posts in Korean.


## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-5635
